src/Symfony/Component/HttpKernel/Log/Logger.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Log;
+
+use Psr\Log\AbstractLogger;
+use Psr\Log\InvalidArgumentException;
+use Psr\Log\LogLevel;
+
+/**
+ * Minimalist PSR-3 logger designed to write in stderr or any other stream.
+ *
+ * @internal
```

Comment:
should be removed

Reply:
Sure? Why not but https://github.com/symfony/symfony/pull/24300#discussion_r141750369

Reply:
Yep, I don't agree this should be internal.
We're writing a framework, we should provide open/closed building blocks, not only closed ones.
Here, there is no reason to make the class neither internal not final: all methods are in a PSR interface.

Reply:
The problem here is the constructor. We're still not very sure about it. But removed, it should be easy to deprecate some arguments if needed.

Reply:
Marking it `@internal` is important IMHO. Not for technical reasons, just to reinforce the idea that this is a simple logger that should not be extended/decorated. If the need is more complex, people should move to Monolog instead.

Reply:
my comment was more about `final`, sorry :)

Reply:
So `final`, `@internal` or both? I vote for both.

Reply:
Internal means we can rename the class as we want, that's not the case to me.
If I want to create a lazy version of this logger, I should be allowed to.
So *at most*, `@final` to me even if I see no good reason it should, as explained.

Reply:
Hmm, after a second though `@internal` doesn't make sens now that we can inject a formatter.

Reply:
Ok, let's remove both @final and @internal.